src/cascadia/TerminalApp/Profile.idl

```diff
@@ -14,6 +17,8 @@ namespace TerminalApp
         Profile();
         Profile(Windows.Foundation.IReference<Guid> guid);
 
+        TerminalSettings CreateTerminalSettings(Windows.Foundation.Collections.IMapView<String, ColorScheme> schemes);

```

Comment:
This may not need to be in the idl if it's a private implementation detail, right?